docs/_documentations/mdt-che-overview.md

```diff
@@ -20,16 +20,5 @@ The Eclipse tools are [open source](https://github.com/eclipse/codewind-eclipse)
 
 To get started, see [Getting started with Codewind](mdteclipsegettingstarted.html).
```

Comment:
The whole mdt-che-overview page doesn't make sense, it is a copy of the Eclipse overview, when Eclipse the IDE has nothing to do with Eclipse che!

But, it appears to be harmless, since the "overview" page for Che is https://www.eclipse.org/codewind/installoncloud.html

I think that mdt-eclipse-overview is not linked to by anything and is a dead page

Reply:
The Codewind for Eclipse Che overview page links to https://www.eclipse.org/codewind/installoncloud.html and `mdt-che-overview` is never used on our landing page before, which doesn't make sense. We should have two separate files for `Overview` and `Installation` under Codewindfor Eclipse Che. That's why I ask Sarah to create this issue: https://github.com/eclipse/codewind/issues/843. Once this PR merges, we should have a better Codewind for Eclipse Che overview page includes introduction and features summary and have a installation page which is similar to the one we have before.

Reply:
@tetchel Noted, and what Jingfu's mentioned above. We're iterating on this to get some consistency for the information we present for each IDE.